.eslintrc

```diff
@@ -126,70 +126,82 @@ rules:
   no-sync: 0
 
   # Stylistic Issues
-  array-bracket-spacing: 0
+  array-bracket-spacing: [2, never]
   block-spacing: 0
-  brace-style: 0
-  camelcase: 0
-  comma-spacing: 0
-  comma-style: 0
-  computed-property-spacing: 0
-  consistent-this: 0
-  eol-last: 0
-  func-names: 0
+  brace-style: [2, 1tbs]
+  camelcase: 2
+  comma-dangle: [2, never]
```

Comment:
Just remembered about [that comment](/chartjs/Chart.js/pull/3201#issuecomment-246359907) and I agree with rjmcguire. Should we allow (but not enforce) trailing comma in this special case (`comma-dangle: [2, only-multiline]`)?